data-otservbr-global/npc/sam.lua

```diff
@@ -210,6 +210,7 @@ npcConfig.shop = {
 	{ itemName = "legion helmet", clientId = 3374, sell = 22 },
 	{ itemName = "longsword", clientId = 3285, buy = 160, sell = 51 },
 	{ itemName = "mace", clientId = 3286, buy = 90, sell = 30 },
+	{ itemName = "magic plate armor", clientId = 3366, sell = 6400 },
```

Comment:
I can't find this item on this npc.

Reply:
Sam is the Blacksmith of [Thais](https://tibia.fandom.com/wiki/Thais). His real name is Samuel, but he prefers to be called Sam. He was named after his grandfather. He sells and buys many weapons and armors. Sam is one of the few [NPCs](https://tibia.fandom.com/wiki/NPC) that buys magic plate armors. The only way to sell it to him is by typing "Sell magic plate armor". 

https://tibia.fandom.com/wiki/Sam

Reply:
Also, on line 228 you can find the studded armor as well..